packages/react/src/components/Icon/icons/check-circle.svg

```diff
@@ -1 +1,8 @@
-<svg version="1.2" xmlns="http://www.w3.org/2000/svg" overflow="visible" preserveAspectRatio="none" viewBox="0 0 24 24" width="20" height="20"><g><path d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm-2 15l-5-5 1.41-1.41L10 14.17l7.59-7.59L19 8l-9 9z" vector-effect="non-scaling-stroke" fill="currentColor" /></g></svg>
\ No newline at end of file
+<svg version="1.2" xmlns="http://www.w3.org/2000/svg" overflow="visible" preserveAspectRatio="none"
+	 viewBox="0 0 44 44" vector-effect="non-scaling-stroke" fill="currentColor">
+<style type="text/css">
+	.st1{fill:#FFFFFF;}
+</style>
```

Comment:
This breaks dark mode: 
![image](https://user-images.githubusercontent.com/6416791/185150632-f461e536-d260-411c-a02f-cb72bb182773.png)

For backwards compatibility and dark mode, we should not fill the check by default. Instead, the radio card group should fill the check by targeting the `polygon` element

Reply:
We may need to think about giving this a default colour, removing all styling: 

<img width="1636" alt="image" src="https://user-images.githubusercontent.com/41127686/185235858-9317eb53-dd1f-47d8-a283-717720042cc9.png">

<img width="886" alt="image" src="https://user-images.githubusercontent.com/41127686/185235955-2fc1b71e-7e5d-488e-8b36-dc4712c15878.png">

Reply:
Gotcha. Alternatively, we could  add a new Icon type "check-circle-solid" (similar to how we have other "solid" icon variants). We would then have light and dark theme stylings for it to fill in the checkmark appropriately.

Although I just noticed `cardIcon` is passed in to the RadioCardGroup as a string, which makes it tricker to do this in a forwards-compatible way.

I'm curious to get @scurker's thoughts on this.

Reply:
I don't think the problem is inherent to the icon here. Cauldron icons don't tend to have a "default" color, but rather inherit the color from the text.

The underlying issue to me seems to be is in how the icon is used and the color. Green is notorious for being difficult to pass color contrast on light backgrounds, so my first suggestion would be to ask if we can apply a darker color for the icon in the RadioCardGroup component.

Reply:
Thoughts on this colour? 

`color: #4f990f` - I suspect we'll need a CSS variable if we go  down right route?

<img width="835" alt="image" src="https://user-images.githubusercontent.com/41127686/185404847-2b31661e-4f6c-4840-9140-9a61b6f73680.png">
<img width="611" alt="image" src="https://user-images.githubusercontent.com/41127686/185405024-eb14cab8-5193-4899-a7b2-ba1dbb2c4abd.png">

Reply:
> my first suggestion would be to ask if we can apply a darker color for the icon in the RadioCardGroup component

Originally, I was trying to suggest a way to avoid introducing another color by making it so the checkmark could be made white. However, after becoming more familiar with how we handle iconography in cauldron, I'm inclined to agree that just using a darker color is the better option. Sorry to lead you astray, @Zidious

Reply:
> Thoughts on this colour?

It looks like it's passing 3:1, although I would double check with @awpearlm to get him to sign off.

Reply:
Its fine to change the color, just make it a one-off rather than some color that's gonna find it's way into Cauldron...at least for now. The Mobile team should be notified of the change so they can change their respective icon to be that color too @karnett @ryomtoob

Reply:
Thanks for bringing this to my attention @awpearlm. The mobile workflow is using the `<RadioCardGroup>` with `cardIcon=check-circle`. https://github.com/dequelabs/walnut/blob/develop/app/src/modules/axe-devtools-mobile/components/ChooseOS.tsx#L30-L53
I think that means when this this PR gets merged, released, and adopted into walnut, it should be fixed for us too.